### ISSUE TITLE
Help wanted: add support to macOS

### ISSUE THREAD

**Issue by kjlawrencelaw** [maple3142/Netease-Music-Proxy#1]
As title, impact on this project just now and figure out this is windows only,
is there any plan to support macOS as well?

**Comment by maple3142** [comment]
No, but perhaps it would work fine with wine running on MacOS. It is just a GUI that setup a proxy adding a specific header to requests for netease, so its core functionality can probably easily achieved by mitmproxy.
